java/test/src/main/java/test/Ice/optional/Test.ice

```diff
@@ -346,8 +336,6 @@ interface Initial
     bool supportsCsharpSerializable();
 
     bool supportsCppStringView();
-
-    bool supportsNullOptional();
```

Comment:
Probably should keep supportsNullOptional() (returns false) for now, for cross-testing.